runtime/src/chplmemtrack.c

```diff
@@ -562,6 +569,23 @@ printMemAllocs(chpl_mem_descInt_t description, int64_t threshold,
       }
     }
   }
+  // allocate the table
+  table = (memTableEntry*)sys_malloc(n*sizeof(memTableEntry));
+  if (!table)
+    chpl_error("out of memory printing memory table", lineno, filename);
+  // save the relevant memTable entries to 'table' to get a snapshot
+  n = 0;
+  for (i = 0; i < hashSize; i++) {
+    for (memEntry = memTable[i]; memEntry != NULL; memEntry = memEntry->nextInBucket) {
+      size_t chunk = memEntry->number * memEntry->size;
+      if (chunk < threshold)
+        continue;
+      if (description != -1 && memEntry->description != description)
+        continue;
+      table[n++] = *memEntry;
+    }
```

Comment:
It is a little weird to me to reuse the table size variable as index variable in this loop. Why not create a new variable that has a more descriptive name for the intended purpose?

Reply:
I realized after writing this is not new code, just moved around. I  think my point still stands, but I  don't think its strictly necessary to change

Reply:
I realized after writing this is not new code, just moved around. I  think my point still stands, but I  don't think its strictly necessary to change